python/ray/autoscaler/_private/kuberay/node_provider.py

```diff
@@ -248,18 +223,18 @@ def create_node(
             url = "rayclusters/{}".format(self.cluster_name)
             raycluster = self._get(url)
             group_name = tags["ray-user-node-type"]
-            group_index, group_spec = self._get_worker_group(raycluster, group_name)
+            group_index, _ = self._get_worker_group(raycluster, group_name)
+            tag_filters = {TAG_RAY_USER_NODE_TYPE: groupName}
```

Comment:
`groupName` -> `group_name` is a typo here I think :)

Reply:
Fixed. Thanks.